.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: ruby
+before_install: gem update --system
```

Comment:
A quick question: Is this necessary?

Reply:
It seems to when using `cache: bundler` and we test a new Ruby version